models.md

```diff
@@ -0,0 +1,35 @@
+# Models definition
++ The store needs to be able to map type names to TS classes.
+
+### Proposition 1
+
+```typescript
+@JsonApiModel({
+  type: 'posts',
+  endpoint: '/posts',
+  baseUrl: 'http://api.example.com'
+})
+class Post {
+  @JsonApiAttribute()
+  title: string
+
+  @JsonApiAttribute()
+  content: string
+
+  @JsonApiAttribute({
+    key: 'published_at'
```

Comment:
could the key be inferred? by the property name? I'm not sure what all these method attributes have access to (I haven't _really_ written any method attributes yet)

The less that has to be repeatedly configured the better. For my project at work, there are likely going to be over 400 models. :-(